example/pubspec.yaml

```
@@ -26,14 +26,14 @@ dependencies:
   flutter_localizations:
     sdk: flutter
 
-  intl: ^0.17.0
+  intl: 0.18.1
```

@mohammadprgrm20123, i've tried to use you branch as source of the package.
I'm facing a problem. It seems that the version that you put ` 0.18.1` is not compatible with `flutter_localizations` package in flutter 3.10.5 which require `intl 0.18.0`.

May be you can change this dependency to `intl: ^0.18.0`

---

Done

---

May be you should put the `^`in of the version to ensure that it will work in future versions.
```suggestion
  intl: ^0.18.0
```

---

Done